src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1608,43 +1614,48 @@ else if (toUpdate.getContentAccessMode() != null) {
      * @return boolean status whether any updates are made or not.
      */
     private boolean updateEnvironment(ConsumerDTO updated, Consumer toUpdate) {
```

Comment:
Let's update the name of the method and javadoc to reflect that this method is now updating multiple environments

Reply:
Done.